internal/blocktx/blockchain_communication/p2p/init.go

```diff
@@ -1,25 +1,22 @@
-package blocktx
+package blocktx_p2p
```

Comment:
I think this package could be called just `p2p`, not?
From the path it is clear that it is for `blocktx`

Reply:
I use the `p2p` package here, so I thought it would be a good idea to distinguish `blocktx_p2p`, `metamorph_p2p`, or `whatever_p2p` from the general `p2p` package. However, I can also use alias for the general package as well

Reply:
In favour of alias

Reply:
I will experiment with this in follow up PR